_examples/idv/handlers.session.go

```diff
@@ -208,3 +212,83 @@ func showPrivacyPolicyPage(c *gin.Context) {
 	render(c, gin.H{}, "privacy.html")
 	return
 }
+
+/* For Testing */
+
+func showDigitalPage(c *gin.Context) {
+	sessionReq, err := buildDigitalIdentitySessionReq()
+	if err != nil {
+		c.HTML(
+			http.StatusInternalServerError,
+			"error.html",
+			gin.H{
+				"ErrorTitle":   "Error when building sessions spec",
+				"ErrorMessage": err.Error()})
+		return
+	}
+	pageFromShareSessionReq(c, sessionReq)
+}
+
+func initialiseDigitalIdentityClient() error {
+	var err error
+	sdkID = os.Getenv("YOTI_CLIENT_SDK_ID")
+	keyFilePath := os.Getenv("YOTI_KEY_FILE_PATH")
+	key, err = os.ReadFile(keyFilePath)
+	if err != nil {
+		return fmt.Errorf("failed to get key from YOTI_KEY_FILE_PATH :: %w", err)
+	}
+
+	didClient, err = yoti.NewDigitalIdentityClient(sdkID, key)
+	if err != nil {
+		return fmt.Errorf("failed to initialise Share client :: %w", err)
+	}
+	//didClient.OverrideAPIURL("https://connect.public.stg1.dmz.yoti.com/share/")
+	didClient.OverrideAPIURL("https://api.yoti.com/share")
+
+	return nil
+}
+
+func pageFromShareSessionReq(c *gin.Context, sessionReq *digitalidentity.ShareSessionRequest) {
+	err := initialiseDigitalIdentityClient()
+	if err != nil {
+		c.HTML(
+			http.StatusUnprocessableEntity,
+			"error.html",
+			gin.H{
+				"ErrorTitle":   "Error initialising DID Client",
+				"ErrorMessage": errors.Unwrap(err)})
+		return
+	}
+	b, err := json.Marshal(sessionReq)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	fmt.Println(string(b))
+
+	shareSession, err := didClient.CreateShareSession(sessionReq)
```

Comment:
Better to split this in 2 commits
`commit_a`: create example for session creation
`commit_b`: add session QR creation to example (this commit should have the SDK-2254 in the name)

Reply:
That example is just testing purposes and now we are removing from idv.